src/app/routes/connector-ui/contract-agreement-page/contract-agreement-cards/contract-agreement-cards.component.html

```diff
@@ -58,18 +59,30 @@
           </div>
         </div>
       </div>
-      <!-- Counterparty Address -->
-      <div class="flex flex-col flex-grow">
-        <div class="text-slate text-sm uppercase tracking-wider">
-          Other Connector
+      <div class="flex flex-row space-x-[10px] justify-between">
+        <!-- Counterparty Address -->
+        <div class="flex flex-col">
+          <div class="text-slate text-sm uppercase">Other Connector</div>
+          <div>
+            <a
+              class="link break-all"
+              target="_blank"
+              [attr.href]="card.counterPartyAddress"
+              >{{ card.counterPartyAddress }}</a
+            >
+          </div>
         </div>
-        <div>
-          <a
-            class="link"
-            target="_blank"
-            [attr.href]="card.counterPartyAddress"
-            >{{ card.counterPartyAddress }}</a
-          >
+        <!-- Status -->
+        <div
+          *ngIf="
+            card.direction === 'CONSUMING' &&
+            ('connector-limits' | isActiveFeature)
+          "
+          class="flex flex-col">
+          <div class="text-slate text-sm uppercase tracking-wider">Status</div>
+          <div class="text-right" [matTooltip]="card.statusTooltipText">
```

Comment:
we should add the tooltip to the entire expression, I think, so also including the title.

We should also add a tiny "info" (literally the code I think) icon, either beind the value or the label (wherever it looks better) as a hint that we have a message / tooltip